pandas/tests/io/test_orc.py

```diff
@@ -10,7 +10,6 @@
 import pandas._testing as tm
 
 pytest.importorskip("pyarrow", minversion="0.13.0")
-pytest.importorskip("pyarrow.orc")
```

Comment:
I think the tests might fail without this, since orc doesn't import properly for certain OSes.

Reply:
Yes, it will raise some errors on Windows env. 

But if add this line to `test_orc.py`, the test case can not find the bug mentioned in #40918, it just skips this test case.

Reply:
Maybe try ```pyarrow._orc```? (That's what it tries to ```pyarrow.orc``` tries to find for me on windows)

Reply:
Sorry, let me re-describe the reason for deleting this line. 

If the PyArrow package is installed from Conda, `pytest.importorskip("pyarrow.orc")` will successfully import pyarrow.orc module, so [test_orc.py-Line143](https://github.com/pandas-dev/pandas/blob/master/pandas/tests/io/test_orc.py#L143) `got = read_orc(inputfile).iloc[:10]` will works fine.

But, AttributeError will be raised if the user uses `pd.read_orc` directly without importing `pyarrow.orc` first. The test case failed to find this bug.

---

Maybe we should keep `pytest.importorskip("pyarrow.orc")` and delete  `pytest.importorskip("pyarrow.orc")`. Then fix this bug in pandas/io/orc.py, and make `pyarrow` be imported only once(discuss below).

Reply:
Well, I am hypothesizing that the bug will reproduce since ```pyarrow.orc``` is not imported by ```pytest.importorskip(pyarrow._orc)```, and running ```pytest.importorskip(pyarrow._orc)``` will skip the test on Windows where pyarrow orc support is not present, as I think it is ```pyarrow._orc``` where the orc stuff is actually implemented. I cannot verify this works, though.